app/src/main/java/at/roteskreuz/stopcorona/screens/menu/MenuController.kt

```diff
@@ -33,7 +38,19 @@ class MenuController(
     private val onRevokeSicknessClick: () -> Unit
 ) : EpoxyController() {
 
-    var ownHealthStatus: HealthStatusData by adapterProperty(HealthStatusData.NoHealthStatus)
+    private var ownHealthStatus: HealthStatusData by adapterProperty(HealthStatusData.NoHealthStatus)
+    private var exposureNotificationPhase: ExposureNotificationPhase? by adapterProperty(null as ExposureNotificationPhase?)
+    private var dateOfFirstMedicalConfirmation: ZonedDateTime? by adapterProperty(null as ZonedDateTime?)
+
+    fun setData(
```

Comment:
why not addressing the defined adapterProperties directly?

Reply:
I forgot to finish the implementation, sorry. 
The point is that adapterProperty automatically calls requestModelBuild on each set.
This should be called only on end of setData method.
Updated.